go.mod

```
@@ -3,7 +3,8 @@ module github.com/falcosecurity/falco-exporter
 go 1.13
 
 require (
-	github.com/falcosecurity/client-go v0.1.1-0.20200529161230-8578c36af411
+	// todo(leogr): update version once client-go with bidi support has been released
+	github.com/falcosecurity/client-go v0.1.1-0.20200609153459-3b6f8eb9e49d
```

```suggestion
	github.com/falcosecurity/client-go v0.2.0
```
TODO: fix when `client-go v0.2.0` has been released, then `go mod tidy`

---

Done by https://github.com/falcosecurity/falco-exporter/pull/33/commits/227315cfbbf2771d570654b6ade8c2110f3eb152